api/proto/teleport/legacy/types/types.proto

```diff
@@ -6213,6 +6215,170 @@ enum PluginStatusCode {
   SLACK_NOT_IN_CHANNEL = 10;
 }
 
+// OktaPluginSyncStatusCode indicates the possible states of an Okta
+// synchronization service.
+enum OktaPluginSyncStatusCode {
+  // STATUS_CODE_UNSPECIFIED is the status code zero value, indicating that the
+  // service has not yet reported a status code.
+  STATUS_CODE_UNSPECIFIED = 0;
```

Comment:
nit: enum fields usually are defined with whole of the enum name. It becomes loong name but that is how other enum protos are define too.  Also applies to success and error fields below. 
```suggestion
  OKTA_PLUGIN_SYNC_STATUS_CODE_UNSPECIFIED = 0;
```

Reply:
Was trying to avoid having to write

```go
status := types.OktaPluginSyncStatusCode_OKTA_PLUGIN_SYNC_STATUS_CODE_SUCCESS
```

... but I guess there's no getting around it :-)